internal/resource.go

```diff
@@ -17,3 +17,13 @@ var ComponentDependencyWildcardMatcher = ResourceMatcher(componentDependencyWild
 type componentDependencyWildcardMatcher string
 
 func (c componentDependencyWildcardMatcher) notActuallyImplementedYet() {}
+
+// SLAMDependencyWildcardMatcher is used internally right now for lack of a better way to
+// "select" slam services that another resource is dependency on. Usage of this is an
+// anti-pattern and a better matcher system should exist.
+var SLAMDependencyWildcardMatcher = ResourceMatcher(slamDependencyWildcardMatcher("*:service:slam/*:*"))
+
+type slamDependencyWildcardMatcher string
+
+// TODO:  @erd is it time to do this now?
```

Comment:
obviously will remove this but was wondering if since we now have a second matcher if we should think about the actual interface.  I was thinking it would look something like `Match(map[Name]Resource) []Resource` or something like that